pkg/branch/setupProw.go

```diff
@@ -28,14 +29,14 @@ func SetupProw(manifest model.Manifest, release string, dryrun bool) error {
 	log.Infof("*** Updating prow config for new branches.")
 	repo := "test-infra"
 
-	cmd := util.VerboseCommand("go", "run", "tools/prowgen/cmd/prowgen/main.go", "branch", release)
-	cmd.Dir = manifest.RepoDir(repo)
+	cmd := util.VerboseCommand("go", "run", "./cmd/prowgen/main.go", "branch", release)
+	cmd.Dir = path.Join(manifest.RepoDir(repo), "tools/prowgen")
 	if err := cmd.Run(); err != nil {
 		return fmt.Errorf("failed to generate new prow config: %v", err)
 	}
 
-	privateCmd := util.VerboseCommand("go", "run", "tools/generate-transform-jobs/main.go", "branch", release)
-	privateCmd.Dir = manifest.RepoDir(repo)
+	privateCmd := util.VerboseCommand("go", "run", "main.go", "branch", release)
```

Comment:
You probably didn't need to update this second one as it still uses the parent go.* files, but for consistency.